docs/03.API-EXAMPLE.md

```diff
@@ -11,7 +11,7 @@ This guide is intended to introduce you to JerryScript embedding API through cre
 int
 main (int argc, char *argv[])
 {
-  const jerry_char_t script[] = "print ('Hello, World!');";
+  const jerry_char_t script[] = "var str = 'Hello, World!';";
```

Comment:
I think there are some missing parts, because this source couldn't generate the described output at [line 26](https://github.com/jerryscript-project/jerryscript/pull/1846/files#diff-baad673062cfc24c82fc89196034cdddR26)